<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyCourseWidget</class>
 <widget class="QWidget" name="ModifyCourseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>74</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>74</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>426</width>
    <height>74</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改课程</string>
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTeacher">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>授课教师</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCap">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>上限人数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTeacher">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbCap">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelCntInfo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已选人数</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCnt">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnConfirm</sender>
   <signal>clicked()</signal>
   <receiver>ModifyCourseWidget</receiver>
   <slot>confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>confirm()</slot>
 </slots>
</ui>
